core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -89,6 +96,8 @@ class ConsumerBounceTest extends AbstractConsumerTest with Logging {
     val producer = createProducer()
     producerSend(producer, numRecords)
 
+    this.consumerConfig.setProperty(ConsumerConfig.MAX_POLL_INTERVAL_MS_CONFIG, "30000")
+
```

Comment:
Is this override still needed for the tests to pass?

Reply:
Ah thanks for the catch. We shouldn't need it. Let me change it back